app/src/componentLibrary/attributeEditor/attributeEditor.jsx

```diff
@@ -216,7 +216,7 @@ export const AttributeEditor = ({
           {Parser(CheckIcon)}
         </div>
         <div
-          tabIndex={0}
+          tabIndex={!isCancelButtonDisabled ? 0 : -1}
```

Comment:
May be just use the bool value as is?
```suggestion
          tabIndex={isCancelButtonDisabled ? -1 : 0}
```